framework/include/vectorpostprocessors/LineMaterialSamplerBase.h

```diff
@@ -179,6 +184,7 @@ LineMaterialSamplerBase<T>::execute()
       addSample(_q_point[qp], qp_proj_dist_along_line, values);
     }
   }
+  _fe_problem.clearActiveMaterialProperties(_tid);
```

Comment:
The material property dependency setup part is a new pattern though - one that seems like it will be easy to get wrong and bite people.